pages/doc/opentracing_to_opentelemetry_migration.md

```diff
@@ -0,0 +1,177 @@
+---
+title: Migrate from OpenTracing to OpenTelemetry
+keywords: data, distributed tracing
+tags: [tracing]
+sidebar: doc_sidebar
+permalink: opentracing_to_opentelemetry_migration.html
+summary: Migrate your Java application from OpenTracing to use OpenTelemetry.
+---
+
+OpenTracing is deprecated. ([OpenTracing](https://opentracing.io/) and [OpenCensus](https://opencensus.io/) have merged to form [OpenTelemetry](https://opentelemetry.io/).) To send trace data to Tanzu observability, use OpenTelemetry.
+
+{% include note.html content="Read the [OpenTelemetry documentation](https://opentelemetry.io/docs/migration/opentracing/) for migration options and strategies." %}
+
+This page shows you how to migrate a Java application that uses OpenTracing to use OpenTelemetry.
+
+## Update the pom.xml File
+
+To add the OpenTelemetry dependencies to your Java project's pom.xml file, follow these steps.
```

Comment:
I think it should be `pom.xml` (in backticks) similarly to the `OTelConfig.java` file.